daemon-scheduler/pkg/api/v1/api_test.go

```diff
@@ -282,7 +283,7 @@ func (suite *APITestSuite) TestDeleteEnvironmentMissingEnvironment() {
 	assert.Equal(suite.T(), http.StatusNotFound, responseRecorder.Code)
 }
 
-func (suite *APITestSuite) assertSame(environment *types.Environment, environmentModel *models.Environment) {
+func (suite *APITestSuite) assertSame(environment *environmenttypes.Environment, environmentModel *models.Environment) {
```

Comment:
Yes for variables. No for packages. environmenttypes is a package, not a variable. You'll see we reference package names in all lowercase everywhere.